Makefile

```diff
@@ -50,6 +50,30 @@ build:							## Build dev docker image
 		--build-arg K6_VERSION=${K6_VERSION} \
 		-f Dockerfile.dev  -t instill/${SERVICE_NAME}:dev .
 
+.PHONY: test-dockerfile
+test-dockerfile:							## Run test container with image built by Dockerfile
```

Comment:
are we sure about the naming? are we testing the dockerfile?

Reply:
You can provide a better name.
How about `test-container`?

Reply:
`run-dev-services`?